Documentation/k8s-pre-reqs.md

```diff
@@ -66,7 +66,34 @@ Environment="RKT_OPTS=--volume modprobe,kind=host,source=/usr/sbin/modprobe \
 ...
 ```
 
-Note that the kubelet also requires access to the userspace `rbd` tool that is included only in hyperkube images tagged `v1.3.6_coreos.0` or later.
+Note that the kubelet also requires access to the userspace `rbd` tool that is included only in hyperkube images tagged `v1.3.6_coreos.0` or later. See next section on how to deploy `rbd`.
+
+### Ceph and RBD utilities installed on the nodes
+
+The Kubernetes kubelet shells out to system utilities to mount Rook volumes. This means that every Kubernetes hosts must have these utilities installed. This requirement extends to the control plane, since there may be interactions between kube-controller-manager and the Ceph cluster.
```

Comment:
i think we're missing one last sentence that gives the user the guidance on where to run these commands.  Something that is explicitly stating to go run these commands on all machines in your cluster.